test/SecTester.Scan.Tests/Commands/UploadHarTests.cs

```diff
@@ -4,7 +4,10 @@ public class UploadHarTests
 {
   private const string HarFileName = "filename.har";
 
-  private static readonly Har Har = new();
+  private static readonly Har Har = new(
+    new Log(
+      new Tool("Configuration_Name", "Configuration_Version"))
```

Comment:
Avoid using any cryptic and weird test data
```suggestion
      new Tool("name", "v1.1.1"))
```